internal/dataplane/parser/translators/httproute_atc_test.go

```diff
@@ -353,21 +356,25 @@ func TestCalculateHTTPRoutePriorityTraits(t *testing.T) {
 		},
 		{
 			name: "wildcard hostname and prefix path",
-			httpRoute: &gatewayv1beta1.HTTPRoute{
-				ObjectMeta: metav1.ObjectMeta{
-					Namespace: "default",
-					Name:      "wildcard-hostname-prefix-path",
-				},
-				Spec: gatewayv1beta1.HTTPRouteSpec{
-					Hostnames: []gatewayv1beta1.Hostname{"*.foo.com"},
-					Rules: []gatewayv1beta1.HTTPRouteRule{
-						{
-							Matches: []gatewayv1beta1.HTTPRouteMatch{
-								builder.NewHTTPRouteMatch().WithPathPrefix("/foo/").Build(),
+			match: SplitHTTPRouteMatch{
+				Source: &gatewayv1beta1.HTTPRoute{
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace: "default",
+						Name:      "wildcard-hostname-prefix-path",
+					},
+					Spec: gatewayv1beta1.HTTPRouteSpec{
+						Hostnames: []gatewayv1beta1.Hostname{"*.foo.com"},
+						Rules: []gatewayv1beta1.HTTPRouteRule{
+							{
+								Matches: []gatewayv1beta1.HTTPRouteMatch{
+									builder.NewHTTPRouteMatch().WithPathPrefix("/foo/").Build(),
```

Comment:
If you only have 1 entry you should be able to use `.IntoSlice()`

Reply:
OK, will do

Reply:
done.